tmt/steps/report/html.py

```diff
@@ -31,6 +31,15 @@ class ReportHtmlData(tmt.steps.report.ReportStepData):
         help='Make paths absolute rather than relative to working directory.'
         )
 
+    display_guest: str = field(
+        default='always',
```

Comment:
@happz isn't `auto` a better default option? I would opt for it

Reply:
In my books, `always` is much better because it's always the same output, no matter how many guests are involved. Let's wait for someone to decide, @psss @lukaszachy care to chime in?

Reply:
I vote for 'always' to have stable column names

Reply:
I'd vote for `auto` as @thrix suggested. Let's make the default view concise if just a single guest is involved.

Reply:
Changed in e69ea8e2

Reply:
I respect the voting ;)

Reply:
I believe this can be now closed.